tools/scripts/identify_and_kill_instances_by_id.py

```diff
@@ -0,0 +1,95 @@
+#!/bin/env python3
+import argparse
+import os
+
+import boto3
+from dotenv import load_dotenv
+from tqdm import tqdm
+
+
+"""
+This script expects a file named instances.txt with one AWS instance id per line.
```

Comment:
Looks like this is no longer the default file its looking for and has become a required argument. Either update  the wording here or make `instances.txt` the default filename so that the docstring matches the code..

Reply:
Thanks for noticing!